streampark-console/streampark-console-service/src/main/assembly/script/data/mysql-data.sql

```diff
@@ -30,7 +30,12 @@ insert into `t_team` values (100000, 'default', null, now(), now());
 -- ----------------------------
 -- Records of t_flink_app
 -- ----------------------------
-insert into `t_flink_app` values (100000, 100000, 2, 4, null, null, 'Flink SQL Demo', null, null, null, null, null, null , null, 100000, null, 1, null, null, null, null, null, null, null, null, 0, 0, null, null, null, null, null, null, 'Flink SQL Demo', 0, null, null, null, null, null, null, null, 0, 0, now(), now(), null, 1, 1, null, null, null, null, null, null, 0, null, null, null, 'streampark,test', 0, null);
```

Comment:
I suggest using `insert table $table($column1,$column2,$column3)
values(...)` instead of `insert into $table values(...)` it is a better solution